tests/reftests/cli-versioning.test

```diff
@@ -61,8 +61,153 @@ opam: option was added in version 2.1 of the opam CLI, but version 2.0 has been
 [ERROR] Field or section foo not found
 ### OPAMCLI=2.0 opam option foo --global
 opam: global was added in version 2.1 of the opam CLI, but version 2.0 has been requested, which is older.
-### # opam lock use mk_command
+### # opam lock uses mk_command
 ### opam lock foo
 [ERROR] No package matching foo
 ### OPAMCLI=2.0 opam lock foo
 opam: lock was added in version 2.1 of the opam CLI, but version 2.0 has been requested, which is older.
+### # Check for build test env
+### # Note: you must have an installed opam with cli version enabled to pass these tests
+### mkdir opams
+### <opams/ok-var-2-0.opam>
+opam-version: "2.0"
+build: [ "opam" "config" "var" "prefix" "--readonly" ]
+### <opams/ok-var-2-0-cli.opam>
+opam-version: "2.0"
+build: [ "opam" "config" "var" "prefix" "--cli=2.0" "--readonly" ]
```

Comment:
```suggestion
### <opams/ok-var-2-0.opam>
opam-version: "2.0"
build: ["sh" "-c" "env | grep -q '^OPAMCLI=2.0$'"]
install: ["sh" "-c" "env | grep -q '^OPAMCLI=2.0$'"]
remove: ["sh" "-c" "env | grep -q '^OPAMCLI=2.0$'"]
### <opams/ok-var-2-0-cli.opam>
opam-version: "2.0"
build-env: [OPAMCLI = "2.1"]
build: ["sh" "-c" "env | grep -q '^OPAMCLI=2.0$'"]
install: ["sh" "-c" "env | grep -q '^OPAMCLI=2.0$'"]
remove: ["sh" "-c" "env | grep -q '^OPAMCLI=2.0$'"]
```

> I've added a test in cli versioning one, can you check that it is exhaustive, what you had in mind to test ?
> I'm not completely convinced with those test as they need an opam system installed with cli versioning, I'd prefer not to depend on that (even if it will be the usual case).

I had more in mind something like that, just to make sure that the environment variable is correctly set.

Reply:
Is it enough to test the presence of the variable in env ? As it is set for opam calls, I tested opam calls. In the other hand, it permit to not rely on a system installed opam requirement...